src/evict/evict_conn.c

```diff
@@ -146,9 +146,9 @@ __evict_validate_config(WT_SESSION_IMPL *session, const char *cfg[])
     if (evict->eviction_updates_trigger < DBL_EPSILON) {
         WT_CONFIG_DEBUG(session,
           "config eviction_updates_trigger (%f) cannot be zero. Setting "
-          "to 50%% of eviction_dirty_trigger (%f).",
-          evict->eviction_updates_trigger, evict->eviction_dirty_trigger / 2);
-        evict->eviction_updates_trigger = evict->eviction_dirty_trigger / 2;
+          "to 100%% of eviction_dirty_trigger (%f).",
```

Comment:
To start with we can have this change only in disagg. For ASC we can consider the incremental roll out